main.go

```diff
@@ -0,0 +1,82 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"cloud.google.com/go/pubsub"
+)
+
+func main() {
+
+	err := publishMessage("pubsub1", "testing1", data{
+		messageID:    "1",
+		requestBody:  "api request body",
+		responseBody: "api response body",
+	})
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+}
+
+type data struct {
+	messageID		string
+	requestBody		string
+	responseBody	string
+}
+
+// PublishMessage publishes messages to already created topics
+func publishMessage(projectID, topicID string, msg data) error {
+	ctx, cancel := context.WithTimeout(context.Background(), 30 * time.Second)
+	defer cancel()
+
+	os.Setenv("PUBSUB_EMULATOR_HOST", "localhost:8085")
```

Comment:
We shouldnt set or get env variables in a library. We should let our caller do this, especially while initializing the cleint.